### PR TITLE
feat(integrations): Use a hatchet on configuredProperties values for now

### PR DESCRIPTION
**Pull request by gashcrumb**
relates to syndesisio/syndesis-rest#451

### CODE REVIEW

File: src/app/integrations/edit-page/step-configure/filter-steps/basic-filter.model.ts [hunk 1]
```diff
@@ -108,10 +108,13 @@ export function createBasicFilterModel(configuredProperties: BasicFilter) {
     ];
   };
   let groups = undefined;
+  let rules = undefined;
   // build up the form array from the incoming values (if any)
   if (configuredProperties && configuredProperties.rules && configuredProperties.rules.length) {
+    // TODO hackity hack
+    rules = JSON.parse(<any>configuredProperties.rules);
```
Comment (rhuss) [inline src/app/integrations/edit-page/step-configure/filter-steps/basic-filter.model.ts]: Not sure whether this is correct in this place, has configuredProperties is of type `BasicFilter` which in turn contains the rules as an array of `Rule` elements already.

**Comment by gashcrumb** [comment]
This bit is for loading the rules *back* from the step, the form expects
this to be an array...

**Comment by rhuss** [comment]
Hmm. But `configuredProperties` is of type `BasicFilter` where rules is an array of `FilterRules`. An actually I indeed got an parse error at this point.

**Comment by gashcrumb** [comment]
`configuredProperties` should be the containing object and `rules` should be the attribute that contains the stringified array.  But lemme check as I couldn't test this yesterday as I couldn't save the filter step still.

**Pull request merged**